app/components/post_list/post/post.tsx

```diff
@@ -57,6 +57,8 @@ type PostProps = {
     testID?: string;
 };
 
+const TIME_TO_FAIL = 10000;
```

Comment:
I like this.

Reply:
Does this mean that we wait 10s before showing the "failed" state on the message? That might be a bit long if so.

Perhaps @matthewbirtch might have thoughts on how long it should take for a failed message send to show as failed

Reply:
I don't have a problem with 10s. It seems like a reasonable time to wait before considering something as 'failed'.